docker/tester-exec.sh

```diff
@@ -15,6 +15,31 @@
 
 # This script relies on workspace which is cleaned via jenkins method
 
+# function to test and image with QEMU
+test_qemu() {
+  wget ${_WGET_OPTS} ${CI_BUILD_URL}/images/${MACHINE}/ovmf.qcow2
+
+  # Make port numbers that won't collide with other test runs
+  PORT1=$((21000+20*${BUILD_NUMBER: -1}+$BUILD_NUMBER+$TEST_ID))
+  PORT2=$((25000+20*${BUILD_NUMBER: -1}+$BUILD_NUMBER+$TEST_ID))
+
+  qemu-system-x86_64 -device virtio-net-pci,netdev=net0,mac=52:54:00:12:35:02 \
```

Comment:
should consider putting some hashed values into MAC address as well?
I think that in default config, net device will not be isolated, (could be isolated with restrict option)
so any concurrent qemu sessions would use colliding MAC address which may lead to trouble.